### PR TITLE
Change test considering CenterContainer

### PR DESCRIPTION
**Pull request by mai291**

### CODE REVIEW

**Review by ManuilovaTA — APPROVED** [review]

File: CaesarLib/LocationPage.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    {
+        private IWebElement groupOfLocations;
+        private IWebElement cityChernivtsy;
+        private IWebElement cityDnipro;
+        private IWebElement cityIvanoFrankivsk;
+        private IWebElement cityKyiv;
+        private IWebElement cityLviv;
+        private IWebElement cityRivne;
+        private IWebElement citySofia;
+        private IWebElement confurmButton;
```
Comment (osmila) [inline CaesarLib/LocationPage.cs:18]: It has to be "confirmButton" (not confurm)

File: CaesarLib/LocationPage.cs [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        }
+
+        public IWebElement ConfurmButton
+        {
+            get
+            {
+                if (confurmButton != null) return confurmButton;
+                else
+                {
+                    confurmButton = driverInstance.FindElement(By.XPath("//i[@class='fa fa-check-circle-o fa-3x']"));
```
Comment (osmila) [inline CaesarLib/LocationPage.cs:139]: Have you tried to locate this element by ClassName("save") ?

File: CaesarLib/LocationPage.cs [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        }
+
+        public IWebElement CancelButton
+        {
+            get
+            {
+                if (cancelButton != null) return cancelButton;
+                else
+                {
+                    cancelButton = driverInstance.FindElement(By.XPath("//i[@class='fa fa-times-circle-o fa-3x']"));
```
Comment (osmila) [inline CaesarLib/LocationPage.cs:152]: Same with ClassName("cancel")

File: CaesarTests/TC_1_05_01.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        [Test]
+        public void ExecuteTest_ChooseLocation_LocationPageOpened()
+        {
+            Actions builder = new Actions(driver);
+            builder.MoveToElement(driver.FindElement(By.ClassName("containerMainMenu"))).Build().Perform();
+            driver.FindElement(By.XPath("//i[@class='fa fa-globe fa-2x']")).Click();
+            locationPageInstance = new LocationPage(driver);
+            Acts.Click(locationPageInstance.CityChernivtsy);
+            Acts.Click(locationPageInstance.ConfurmButton);
+            string exeptualResultTitle = "Chernivtsy";
```
Comment (osmila) [inline CaesarTests/TC_1_05_01.cs:41]: There isn't such word as exeptual) In this case I think you meant "expected"

**Review by osmila — COMMENTED** [review]
I left some minor comments :) Please view them

**Pull request merged**